src/testing/testNamespaces.json

```diff
@@ -0,0 +1,24 @@
+{
+  "ns1": {
+    "title": "title from namespace 1"
+  },
+  "ns2": {
+    "title": "title from namespace 2"
+  },
+  "": {
+    "search": {
+      "label": "Search"
+    },
+    "reviews": "{{count}} review",
+    "reviews_plural": "{{count}} reviews",
+    "boardBasis": {
+      "code": "Board Basis",
+      "code_AI": "All Inclusive"
```

Comment:
From what I've seen, we're mostly avoiding examples from our real sites in phrasebook. This example seems fine to share, but maybe it would set a bad precedence.

Reply:
Ah, I see this example was moved from elsewhere.